view/frontend/web/js/view/payment/method-renderer/ach.js

```diff
@@ -0,0 +1,226 @@
+define(
+    [
+        'Magento_Checkout/js/view/payment/default',
+        'ko',
+        'jquery',
+        'braintree',
+        'braintreeDataCollector',
+        'braintreeAch',
+        'Magento_Braintree/js/form-builder',
+        'Magento_Ui/js/model/messageList',
+        'Magento_Checkout/js/action/select-billing-address',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/model/quote',
+        'mage/translate'
+    ],
+    function (
+        Component,
+        ko,
+        $,
+        braintree,
+        dataCollector,
+        ach,
+        formBuilder,
+        messageList,
+        selectBillingAddress,
+        fullScreenLoader,
+        quote,
+        $t
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                deviceData: null,
+                paymentMethodNonce: null,
+                template: 'Magento_Braintree/payment/ach',
+                achInstance: null,
+                routingNumber: ko.observable(""),
+                accountNumber: ko.observable(""),
+                accountType: ko.observable("checking"),
+                ownershipType: ko.observable("personal"),
+                firstName: ko.observable(""),
+                lastName: ko.observable(""),
+                businessName: ko.observable(""),
+                hasAuthorization: ko.observable(false),
+                business: ko.observable(false), // for ownership type
+                personal: ko.observable(true) // for ownership type
+            },
+
+            clickAchBtn: function () {
+                if (!this.validateForm('#' + this.getCode() + '-form')) {
+                    return;
+                }
+
+                fullScreenLoader.startLoader();
+
+                var self = this;
+
+                var billingAddress = quote.billingAddress();
+
+                let regionCode;
```

Comment:
Are we okay with the lack of opera mini support for let? Probably, but just wanted to check

Reply:
I think we can handle any backlash from Opera Mini fanatics ;)